types/node-zendesk/index.d.ts

```diff
@@ -0,0 +1,659 @@
+        type?: Tickets.Type | null; // Anonymous requests can set type, Authenticated requests cannot
+        custom_fields?: Tickets.Field[] | null;
+        fields?: Tickets.Field[] | null;
+        due_at?: string | null; // Anonymous requests can set due date as long as type == task. Authenticated requests cannot
+        ticket_form_id?: number | null;
+        recipient?: string | null;
+        collaborators?: ZendeskID[] | string[] | Collaborator[];
+    }
+
+    /**
+     * @see {@link https://developer.zendesk.com/rest_api/docs/support/requests#update-request|Zendesk Requests Update}
+     */
+    interface UpdateModel {
+        comment?: Comments.CreateModel;
+        solved?: boolean;
+        additional_collaborators?: ZendeskID[] | string[] | Collaborator[];
+    }
+
+    /**
+     * @see {@link https://developer.zendesk.com/rest_api/docs/support/requests#json-format|Zendesk Requests JSON Format}
+     */
+    interface ResponseModel extends AuditableModel {
+        readonly url: string;
+        readonly subject: string;
+        readonly description: string;
+        readonly status: Tickets.Status;
+        readonly priority: Tickets.Priority | null;
+        readonly type: Tickets.Type | null;
+        readonly custom_fields: Tickets.Field[] | null;
+        readonly fields: Tickets.Field[] | null;
+        readonly organization_id: ZendeskID | null;
+        readonly requester_id: ZendeskID;
+        readonly assignee_id: ZendeskID | null;
+        readonly group_id?: ZendeskID | null;
+        readonly collaborator_ids: ZendeskID[];
+        readonly email_cc_ids: ZendeskID[];
+        readonly via: Tickets.Via;
+        readonly is_public: boolean;
+        readonly due_at: string | null;
+        readonly can_be_solved_by_me?: boolean;
+        readonly solved?: boolean;
+        readonly ticket_form_id?: number | null;
+        readonly recipient: string | null;
+        readonly followup_source_id: string | null;
+    }
+
+    interface RequesterAnonymous {
+        name: string;
+        email?: string;
+        locale_id?: ZendeskID;
+    }
+
+    interface Collaborator {
+        name?: string;
+        email: string;
+    }
+
+    interface CreatePayload {
+        readonly request: CreateModel;
+    }
+
+    interface UpdatePayload {
+        readonly request: UpdateModel;
+    }
+
+    interface ResponsePayload {
+        readonly request: ResponseModel;
+    }
+
+    interface ListPayload extends PaginablePayload {
+        readonly requests: ReadonlyArray<ResponseModel>;
+    }
+
+    namespace Comments {
+        interface CreateModel {
+            url?: string;
+            request_id?: number;
+            body?: string;
+            html_body?: string;
+            public?: boolean;
+            author_id?: ZendeskID;
+            uploads?: ReadonlyArray<string>;
+        }
+
+        // TODO: support VoiceComment model @jgeth 2019-01-22
```

Comment:
It'd probably be wise to notate all the TODOs in one place (probably near the top of the file).

For any types that are missing, it might be wise to expose them as either `any` or `unknown`.

Reply:
Resolved this sole TODO